atlas-cassandra/src/main/java/org/atlasapi/content/v2/serialization/setters/EpisodeSetter.java

```
@@ -52,9 +61,35 @@ public void deserialize(org.atlasapi.content.Content content, Content internal)
 
         Map<Ref, SeriesRef> seriesRefs = internal.getSeriesRefs();
         if (seriesRefs != null && !seriesRefs.isEmpty()) {
-            if(seriesRefs.size() > 1) {
-                log.error("Episode with id {} has more than one series ref", internal.getId());
+
+            if (seriesRefs.size() > 1) {
+                log.error("Episode with id {} has more than one series ref", episodeId);
+                try {
+                    pdIntegrationKey = System.getenv("PD_INTEGRATION_KEY"); // NPE if missing
+                    pdRaiseIncidents = Boolean.parseBoolean(System.getenv("PD_RAISE_INCIDENTS"));
+
+                    PagerDutyClientWithKey pagerDutyClient = new PagerDutyClientWithKey(
+                            PagerDutyEventsClient.create(),
+                            pdIntegrationKey,
+                            pdRaiseIncidents
+                    );
+                    Payload payload = Payload.Builder.newBuilder()
+                            .setSummary(
+                                    String.format(
+                                            "Episode with id %s has more than one series ref",
+                                            episodeId
+                                    )
+                            )
+                            .setSource("atlas-deer")
+                            .setSeverity(Severity.WARNING)
+                            .setTimestamp(OffsetDateTime.now())
+                            .build();
+                    pagerDutyClient.trigger(payload, String.valueOf(episodeId));
+                } catch (NullPointerException e) {
```

Just catch all exceptions here